### PR TITLE
ci: unblock unstable deployments

### PR DESCRIPTION
**Pull request by iartemiev**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by amhinson — APPROVED** [review]
LGTM 👍

File: .circleci/config.yml [hunk 1]
```diff
@@ -660,17 +660,13 @@ jobs:
...
-              npm whoami
-              git config --global user.email $GITHUB_EMAIL
-              git config --global user.name $GITHUB_USER
-              git status
-              git --no-pager diff
-              yarn publish:$CIRCLE_BRANCH
-            else
-              echo "Skipping deploy."
-            fi
+            echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
```
Comment (sammartinez) [inline .circleci/config.yml:663]: Why are we echoing this? Will this stay redacted?

Comment (iartemiev) [reply .circleci/config.yml:663]: We're not echoing it out to the console. We're using that to write it to the file `~/.npmrc`

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

**Pull request merged**